.flake8

```diff
@@ -35,13 +35,16 @@ exclude =
     cfgov/jobmanager/migrations/*.py,
     cfgov/v1/migrations/*.py,
     cfgov/ask_cfpb/migrations/*.py,
+    cfgov/selfregistration/migrations/*.py,
+    cfgov/agreements/migrations/*.py,
 
     # Our settings files might need to conform to different readability
     # standards
     cfgov/cfgov/settings/*,
 
     # Now we come to the bits we exclude because they haven't had a
     # proper cleanup yet.
-    cfgov/v1/tests
+    cfgov/v1/tests,
+    cfgov/agreements/features
```

Comment:
Trivial: If the format of this file supports it, add a comma here so you don't have to edit this line next time we add something to this list.

Reply:
I'll look into that